### PR TITLE
Create ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60006](https://cumulocity.atlassian.net/browse/MTM-60006)
Original PR: [6713](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6713)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: split cockpit into module federation plugins (#6713)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Cockpit split into module federation plugins
```

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60006
+version: 1020.7.0
+---
+split cockpit into module federation plugins (#6713)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:17]: The Claude AI suggested to use the following body:
```suggestion
As part of the Cumulocity IoT platform, the Cockpit application provides a user interface for monitoring and controlling connected devices. To improve the modularity and maintainability of the Cockpit codebase, it has been split into separate module federation plugins. This change allows for better isolation and independent development of Cockpit features, enabling faster iteration and easier customization. Existing Cockpit functionality remains unchanged from a user perspective, ensuring a seamless transition to the new architecture.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md (hunk 2 above)
Comment (reey) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:17]:
```suggestion
The Cockpit application was splitted into further plugins. This allows admins to tailor their cockpit application to their needs without involving any coding efforts. They can remove certain plugins and therefore features which they do not want in their cockpit application. The plugins that have now been made available are: `Cockpit widgets`, `Cockpit alarms`, `Sensor phone`, `Child devices`, `Assets navigator`, `Data point library`, `Bookmarks`, `Location`, `Add location` & `Search`.
```

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md (hunk 1 above)
Comment (reey) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:3]:
```suggestion
title: Pluginification of Cockpit
```

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60006
+version: 1020.7.0
+---
+The Cockpit application was splitted into further plugins. This allows admins to tailor their cockpit application to their needs without involving any coding efforts. They can remove certain plugins and therefore features which they do not want in their cockpit application. The plugins that have now been made available are: `Cockpit widgets`, `Cockpit alarms`, `Sensor phone`, `Child devices`, `Assets navigator`, `Data point library`, `Bookmarks`, `Location`, `Add location` & `Search`.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:17]:
```suggestion
The Cockpit application has been split into further plugins. This allows admins to tailor the Cockpit application to their needs without any coding effort. They can now easily remove certain plugins and thus certain features which they do not want to include in their Cockpit application. The following plugins are now available: 

* Cockpit widgets
* Cockpit alarms
* Sensor phone
* Child devices
* Assets navigator
* Data point library
* Bookmarks
* Location
* Add location
* Search
```

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+date: ""
+title: Pluginification of Cockpit
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:3]:
```suggestion
title: Cockpit application split into plugins
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag Please approve

**Review by rahultsag — CHANGES_REQUESTED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* Cockpit widgets
+* Cockpit alarms
+* Sensor phone
+* Child devices
+* Assets navigator
+* Data point library
+* Bookmarks
+* Location
+* Add location
+* Search
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1020-7-0-split-cockpit-into-module-federation-plugins-6713.md:28]: I will merge now and add your suggestion directly in the source file.

**Pull request merged**